src/main/php/CommandLine/Commands/StaticCodeAnalysis/AllToolsCommand.php

```diff
@@ -37,7 +37,7 @@ public function execute(InputInterface $input, OutputInterface $output)
         $commands = $this->getApplication()->all('sca');
 
         foreach ($commands as $command) {
-            if ($command->getName() === 'sca:all') {
+            if ($command->getName() === 'sca:all' || $command->getName() === 'sca:stan') {
```

Comment:
Bitte lass uns darüber mal sprechen. Bis dahin: Bitte nicht!

Reply:
Alles klar, wie besprochen habe ich wieder reingenommen. Leider werden echt viele Errors schon im coding-standard scan geworfen. Ich würde hier ein  "phpstan-baseline.neon" im Projekt zu generieren empfehlen, um dann den aktuell Stand erstmal hinzunehmen. Dann Level auf 5 setzen für neuen Code.

"phpstan-baseline.neon"  generieren geht so:
``
vendor/bin/phpstan analyse --level 5  --configuration "%DIRTOCONFIG%phpstan.neon"  SCANDIR --generate-baseline 
``
Dieses Baseline-File würde dann von der config eingebunden und würde die Fehler erstmal von derzeitigen Basis ignorieren. Was meinst Du? 

Ich denke, wenn in coding-standard selbst schon bei Level 5 107 Fehler gefunden wurden ,wird es bei den anderen mehr.